README.md

```diff
@@ -1,58 +1,68 @@
 # Project - EdEHR
-> Educational Electronic Health Record system: a [BCcampus](https://bccampus.ca) project
+> Educational Electronic Health Record system: a [BCcampus](https://bccampus.ca) project.
 
-## Description
+This project aims to create a student-focused Eletronic Health System where students and professors would be able to keep track of their 
+assessments. For more information on the project and it's guidelines, please refer to the [About](./about.md) section.
 
-EdEHR is an open source web application developed to teach medical students how to use an EHR before they enter the workforce. It can be accessed only through a learning management system (LMS) and requires a technical person to set it up. Once set up, users can access the system either as an instructor or a student. Credentials to try out either role are included below.
```

Comment:
We need this description here. 
Drop the last line "Credentials .."
It is not correct to say "students and professors would be able to keep track of their assessments".  We say "instructors" rather than "professors".  We can use "students" or "learners" interchangeably. Instructors can evaluate their student's work by providing comments within the EdEHR for the students to read.